Fix:
shout.c:305:2: error: implicit declaration of function 'urlencode' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
--- shout.c.orig	2002-04-24 10:26:40.000000000 -0500
+++ shout.c	2021-03-10 18:12:35.000000000 -0600
@@ -15,6 +15,7 @@
 #define WIN32_LEAN_AND_MEAN
 #endif
 
+#include "encode.h"
 #include "shout.h"
 #include "sock.h"
 #include "timing.h"
